contributing/chestertons-fence.md

```diff
@@ -80,7 +80,8 @@ _Note: We're actually solving this by running the tests server-side - a large an
 
 ## Anything else?
 
-Two things:
+Three things:
```

Comment:
We could now say "A few things" and it would be a little more "evergreen".

Reply:
I like the drama of "Three things" for now 😉